pymchelper/flair/common/log.py

```diff
@@ -62,9 +62,9 @@
 logger = logging.getLogger(__name__)
 
 
-def set_log(l):
+def set_log(log):
```

Comment:
could `log` cause collision with keywords?

Reply:
Nope, `log` is not a keyword